modules/security-context-constraints-about.adoc

```diff
@@ -75,6 +75,16 @@ This SCC allows host file system access as any UID, including UID 0. Grant with
 If additional workloads are run on control plane hosts, use caution when providing access to `hostnetwork`. A workload that runs `hostnetwork` on a control plane host is effectively root on the cluster and must be trusted accordingly.
 ====
 
+|`hostnetwork-v2`
+| Like the `hostnetwork` SCC, but with the following differences:
+
+* `ALL` capabilities are dropped from containers.
+* The `NET_BIND_SERVICE` capability can be added explicitly.
+* `seccompProfile` is set to `runtime/default` by default.
+* `allowPrivilegeEscalation` must be unset or set to `false` in security contexts. 
+
+By default, this SCC replaces the `hostnetwork` SCC for new clusters.
```

Comment:
I did not see this before. Please remove in all three sections. It sounds like the old SCC no longer exists which is not true.